.yarnrc

```diff
@@ -1,4 +1,4 @@
 disturl "https://electronjs.org/headers"
-target "19.1.9"
+target "22.3.0"
```

Comment:
By now `22.3.1` is out.

Reply:
Updated in  https://github.com/microsoft/vscode/pull/166430/commits/6175b0441a6336ebbf6b8c9cfde6d6eb97868811